design_system/lib/src/alert/alerts.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:design_system/design_system.dart';
+import 'package:design_system/src/alert/alert_widget.dart';
+import 'package:flutter/material.dart';
+
+class Alerts {
+  static void showSucess(BuildContext context, String message) {
+    SnackBar snackBar = AlertWidget(
+            color: Theme.of(context).extension<CoopartilharColors>()!.white,
```

Comment:
é bom evitar a exclamação, sugiro usar 
```suggestion
            color: CoopartilharColors.of(context).white,
```